src/renderer/coremods/recovery/styles.css

```
@@ -0,0 +1,19 @@
+[class^="wrapper"] [class^="buttons_"] {
+  flex-direction: column;
+  align-items: center;
+}
+[class^="wrapper"] [class^="button"] {
+  width: 400px;
+  margin: 5px;
+}
+
+.recovery-button {
```

```suggestion
.replugged-recovery-button {
```
The class names that replugged uses are always prefixed with `replugged` for consistency it would be better to do it here too.